telemetry/README.md

```diff
@@ -0,0 +1,70 @@
+# Conjur telemetry quickstart
+
+In order to consume the telemetry data collected by Conjur, you need to setup a Prometheus node to scrape the
+`/metrics` endpoint in Conjur. You can then use the myriad of tools that work on top of Prometheus to visualise the data. 
+
+Here, we provide a quickstart that sets up Prometheus and Graphana to allow you to quickly explore visualisation of the telemetry data collected by Conjur. To be comprehensive in our demostration, this quickstart additionally provides examples of gathering of metrics from services adjacent to Conjur such as Postgres and metrics about the node where Conjur is running. 
```

Comment:
```suggestion
Here, we provide a quickstart that sets up Prometheus and Grafana to allow you to quickly explore visualisation of the telemetry data collected by Conjur. To be comprehensive in our demostration, this quickstart additionally provides examples of gathering of metrics from services adjacent to Conjur such as Postgres and metrics about the node where Conjur is running. 
```

Same typo appears throughout this document.

Reply:
Yikes. I wonder if my mind is gravitating towards graphene or something 😅. I'll fix it